build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
+  echo "Checking if html-build is up to date..."
+  ORIGIN_URL=$(git remote get-url origin)
+  git fetch -q $ORIGIN_URL master
```

Comment:
Because this is fetching $ORIGIN_UR` and not origin, it doesn't update any refs other than FETCH_HEAD. It will print something like this:
```
From github.com:whatwg/html-build
 * branch            master     -> FETCH_HEAD
```

I think this is useless to show by default, but we should omit the -q if VERBOSE I think. (Will update when we agree on behavior.)

If you're concerned about changing FETCH_HEAD, we could use a hidden ref like bisect and rerere does, perhaps refs/html-build/origin/master?